diagrams/onprem/network.py

```diff
@@ -119,6 +119,8 @@ class Vyos(_Network):
 class Wildfly(_Network):
     _icon = "wildfly.png"
 
+class Yarp(_Network):
+    _icon = "yarp.png"
```

Comment:
Don't forget newlines above and below.

```suggestion

class Yarp(_Network):
    _icon = "yarp.png"

```